lib/sendcloud/operations/cancel_parcel_request.rb

```diff
@@ -0,0 +1,32 @@
+module Sendcloud
+  module Operations
+    class CancelParcelRequest < Operation
+      protected
+
+      def http_method
+        :post
+      end
+
+      def endpoint
+        "parcels/#{parcel_id}/cancel"
+      end
+
+      def parcel_id
+        options[:parcel_id]
+      end
+
+      private
+
+      def handle_error(body)
+        case response.status
+        when 400
+          raise BadRequestError.new(response: response, body: body)
+        when 410
+          raise DeletedError.new(response: response, body: body)
+        else
```

Comment:
Debatable whether this should be thrown as an error? Whilst it is an error status code, is this not the expected outcome of a `Create/Delete` request. Just wondering how we will treat this `bloomandwild` side, as surely some will just be deleted rather than cancelled and that's the expected outcome?